tuned/daemon/controller.py

```diff
@@ -252,6 +254,27 @@ def profile_info(self, profile_name, caller = None):
 			profile_name = self.active_profile()
 		return tuple(self._daemon.profile_loader.profile_locator.get_profile_attrs(profile_name, [consts.PROFILE_ATTR_SUMMARY, consts.PROFILE_ATTR_DESCRIPTION], [""]))
 
+	@exports.export("s", "b")
+	def profile_delete(self, profile, caller = None):
```

Comment:
I think it may be better to name it "user_profile_delete" because it operates only on user profiles.